CHANGELOG.md

```diff
@@ -33,6 +33,139 @@ All sections are explained on the link above, they are all optional, and each of
 
 <!-- New entries go after this line -->
 
+## v2.5.0 - 2017-10-11
+
+For more details, [see the full changelog](https://github.com/thelounge/lounge/compare/v2.4.0...v2.5.0) and [milestone](https://github.com/thelounge/lounge/milestone/22?closed=1).
+
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@ DESCRIPTION, ANNOUNCEMENT, ETC. @@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+### Added
+
+- Status message condensing ([#759](https://github.com/thelounge/lounge/pull/759), [#1421](https://github.com/thelounge/lounge/pull/1421) by [@YaManicKill](https://github.com/YaManicKill), [#1437](https://github.com/thelounge/lounge/pull/1437), [#1451](https://github.com/thelounge/lounge/pull/1451), [#1475](https://github.com/thelounge/lounge/pull/1475), [#1485](https://github.com/thelounge/lounge/pull/1485) by [@xPaw](https://github.com/xPaw), [#1417](https://github.com/thelounge/lounge/pull/1417), [#1442](https://github.com/thelounge/lounge/pull/1442), [#1509](https://github.com/thelounge/lounge/pull/1509), [#1524](https://github.com/thelounge/lounge/pull/1524) by [@astorije](https://github.com/astorije))
+- Use `.lounge_home` to help distribution packages handle config paths right ([#1416](https://github.com/thelounge/lounge/pull/1416), [#1587](https://github.com/thelounge/lounge/pull/1587) by [@xPaw](https://github.com/xPaw), [#1418](https://github.com/thelounge/lounge/pull/1418) by [@astorije](https://github.com/astorije))
+- Implement push notifications ([#1124](https://github.com/thelounge/lounge/pull/1124), [#1445](https://github.com/thelounge/lounge/pull/1445), [#1572](https://github.com/thelounge/lounge/pull/1572), [#1468](https://github.com/thelounge/lounge/pull/1468) by [@xPaw](https://github.com/xPaw), [#1463](https://github.com/thelounge/lounge/pull/1463) by [@astorije](https://github.com/astorije))
+- Set default `/quit` message ([#1448](https://github.com/thelounge/lounge/pull/1448) by [@xPaw](https://github.com/xPaw))
+- Gracefully quit on <kbd>Ctrl</kbd> <kbd>C</kbd> ([#1477](https://github.com/thelounge/lounge/pull/1477) by [@xPaw](https://github.com/xPaw))
```

Comment:
Add a `+`

Reply:
Yeah I started with that, then thought it was ugly so removed it. But least important detail in the world and fine either way so I re-added it 😅